front/app/containers/Admin/settings/messages.ts

```diff
@@ -32,6 +32,10 @@ export default defineMessages({
     id: 'app.containers.AdminPage.SettingsPage.tabInputStatuses',
     defaultMessage: 'Input statuses',
```

Comment:
I think in the long term "input statuses" makes more sense than "idea statuses" (or maybe "ideation statuses"), but since the current solution is not yet unified and we have separate sets for proposals and ideation, perhaps we should have the copy "idea statuses" for now?

Reply:
Also had a question on this while looking at the code ^ :)

Reply:
I think it's good to go with Input Statuses. We are using the term input everywhere, since 'ideas' is just one of the terms used for it. Even it were only ideation, we should still call it inputs.